_data/projects/learn-js.yml

```
@@ -0,0 +1,12 @@
+name: LearnJS
+desc: A website having collection of javascript programs and questions. 
+site: https://github.com/akulsr0/learn-js
+tags:
+  - javascript
+  - programs
+  - questions
+  - algorithm
+  - data-structures
+upforgrabs:
+  name: up for grabs
```

Last change to make - this needs to be the same as the label used in the project

```suggestion
  name: help wanted
```